site/src/pages/api/roadmap.ts

```diff
@@ -0,0 +1,81 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import fetch from "node-fetch";
+
+const handler = async (req: NextApiRequest, res: NextApiResponse) => {
+  try {
+    const org = "jpmorganchase";
+    const repo = "salt-ds";
+
+    const query = `
+      query($org: String!, $repo: String!, $endCursor: String) {
```

Comment:
this is cool++ writing graphql queries is not the easiest, especially when using someone else's endpoint